src/components/ui/PickInput/PickInput.tsx

```diff
@@ -2,14 +2,22 @@ import React from 'react'
 import {Input, Icon} from 'antd'
 import {useTranslation} from 'react-i18next'
 import styles from './PickInput.less'
+import {Store} from '../../../interfaces/store'
+import {buildBcUrl} from '../../..'
+import {connect} from 'react-redux'
 
-export interface PickInputProps {
+export interface PickInputOwnProps {
     disabled?: boolean,
     value?: string,
     onClick?: () => void,
     onClear?: () => void,
     className?: string,
     placeholder?:  string
+    popupBcName?: string, // TODO: not optional in 2.0
```

Comment:
I think an optional `spin?: boolean` will work better here as `PickInput` component will stay a simple ui control that way.

Reply:
fixed as `loading?: boolean` option